src/components/About/WarningBanner.js

```diff
@@ -41,8 +41,9 @@ const WarningBanner = ({
   const incompatibleModulesCount = incompatibleModules.length;
   const missingModulesMsg = <FormattedMessage id="stripes-core.about.missingModuleCount" values={{ count: missingModulesCount }} />;
   const incompatibleModuleMsg = <FormattedMessage id="stripes-core.about.incompatibleModuleCount" values={{ count: incompatibleModulesCount }} />;
+  const isLoading = Boolean(missingModulesCount || incompatibleModulesCount);
```

Comment:
hmm, it looks unclear to me, how count > 0 is indicating that something is loading